TestPantsDaemonIntegration.test_pantsd_memory_usage is flaky

It timed out, exceeding the 540s limit: https://travis-ci.org/pantsbuild/pants/jobs/574692133

```
                   Invalidated 1 target.
06:48:39 28:13       [run]
                     
                     Warning: Timeout for tests/python/pants_test/pantsd:pantsd_integration (900s) exceeds 540s. Capping.Only executing 4 of 33 total tests in shard 7 of 20
                     ..F.                                       [100%]
                     ==================== FAILURES ====================
                     _ TestPantsDaemonIntegration.test_pantsd_memory_usage _
                     
                     self = <pants_test.pantsd.test_pantsd_integration.TestPantsDaemonIntegration testMethod=test_pantsd_memory_usage>
                     
                         @unittest.skipIf(sys.version_info[0] == 2,
                                          reason='Increases by ~0.52 under python 2 as described in '
                                                 'https://github.com/pantsbuild/pants/issues/7761.')
                         def test_pantsd_memory_usage(self):
                           """Validates that after N runs, memory usage has increased by no more than X percent."""
                           number_of_runs = 10
                           max_memory_increase_fraction = 0.40 # TODO https://github.com/pantsbuild/pants/issues/7647
                           with self.pantsd_successful_run_context() as (pantsd_run, checker, workdir, config):
                             cmd = ['filter', 'testprojects::']
                             self.assert_success(pantsd_run(cmd))
                             initial_memory_usage = checker.current_memory_usage()
                             for _ in range(number_of_runs):
                     >         self.assert_success(pantsd_run(cmd))
                     
                     .pants.d/pyprep/sources/7fccf78888dd614cb451e52af62446e127ec6e7b/pants_test/pantsd/test_pantsd_integration.py:407: 
                     _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
```

--- a/src/python/pants/pantsd/pantsd_integration_test.py
+++ b/src/python/pants/pantsd/pantsd_integration_test.py
@@ -445,8 +445,6 @@ class TestPantsDaemonIntegration(PantsDaemonIntegrationTestBase):
             # Remove the pidfile so that the teardown script doesn't try to kill process 9.
             os.unlink(pidpath)
 
-    @pytest.mark.skip(reason="flaky: https://github.com/pantsbuild/pants/issues/8193")
-    @pytest.mark.no_error_if_skipped
     def test_pantsd_memory_usage(self):
         """Validates that after N runs, memory usage has increased by no more than X percent."""
         number_of_runs = 10
@@ -462,11 +460,9 @@ class TestPantsDaemonIntegration(PantsDaemonIntegrationTestBase):
                 ctx.checker.assert_running()
 
             final_memory_usage = ctx.checker.current_memory_usage()
-            self.assertTrue(
-                initial_memory_usage <= final_memory_usage,
-                f"Memory usage inverted unexpectedly: {initial_memory_usage} > {final_memory_usage}",
+            assert initial_memory_usage > 0, (
+                f"Expected non-zero initial memory usage, got {initial_memory_usage}"
             )
-
             increase_fraction = (float(final_memory_usage) / initial_memory_usage) - 1.0
             self.assertTrue(
                 increase_fraction <= max_memory_increase_fraction,

